roles/statuscake_monitoring/defaults/main.yml

```diff
@@ -1,5 +1,14 @@
+# Uptime tests defaults
 statuscake_api_key: ""
 statuscake_default_check_rate: 300
 statuscake_default_test_type: HTTP
 statuscake_uptime_tests: []
 statuscake_merged_uptime_tests: []
+
+# SSL tests defaults
+statuscake_ssl_default_check_rate: 86400
+statuscake_ssl_default_alert_at: [1, 2, 3]
```

Comment:
> Comma separated list representing when alerts should be sent (days). Must be exactly 3 numerical values

What does `alert_at` actually mean? Does a value of `10` mean an alert will be sent 10 days before expiration?

Reply:
From the documentation:
_"Comma separated list representing when alerts should be sent (days). Must be exactly 3 numerical values"_

I don't like the vagueness, and unfortunately there is no example.